firestore/src/ios/credentials_provider_desktop.cc

```diff
@@ -18,9 +19,35 @@ using auth::CredentialChangeListener;
 using auth::Token;
 using auth::TokenListener;
 using auth::User;
+using firebase::auth::AuthError;
 using util::Status;
 using util::StatusOr;
 
+Error ConvertAuthErrorToFirestoreError(int e) {
```

Comment:
Consider renaming `ConvertAuthErrorToFirestoreError` to `FirestoreErrorFromAuthError`.

IMO, this makes code at the call site more naturally readable:
```
Error firestoreError = ConvertAuthErrorToFirestoreError(code);
```
vs.
```
Error firestoreError = FirestoreErrorFromAuthError(code); 
```
See go/python-tips/054 for further rationale.

Reply:
Done